client/src/Components/App/App.js

```diff
@@ -69,7 +69,7 @@ const App = () => {
           if (!journal) {
             return null;
           }
-          return <JournalEntry journal={journal}/>
+          return <JournalEntry journal={journal} entryID={id}/>
```

Comment:
okay nice way for us to have the journal and know which entry it is associated with later for editing